warrior/WarriorCore/Classes/argument_datatype_class.py

```diff
@@ -64,27 +65,32 @@ def convert_arg_to_datatype(self):
             self.datatype = tuple
         elif self.arg_name.startswith('dict_'):
             self.datatype = dict
+        elif self.arg_name.startswith('file_'):
+            self.datatype = file
+            try:
+                self.arg_value = file(self.arg_value)
+            except IOError:
+                pass
```

Comment:
I suggest to print an warning message to user that the file is not available in the given location as we are expecting IOError here.